src/NuGet.Clients/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -21,6 +22,10 @@
     <PackageReference Include="SharpZipLib" ExcludeAssets="all" /> <!-- dependency of Lucene.net. Can delete when Lucene.net has an update with a newer dependency on sharpziplib -->
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="docs\README.md" Pack="true" PackagePath="\"/>
```

Comment:
Try adding this relative to `MSBuildProjectFullPath`. 

I'd be happy to help offline.

Reply:
Thanks! Moved the item from individual project files to common.targets.